org.civicrm.bootstrapcivihr/scss/pages/vacancies.scss

```diff
@@ -0,0 +1,315 @@
+    color: $crm-white;
+    font-family: $font-family-base;
+    padding: 8px 12px;
+    text-shadow: none;
+    text-transform: uppercase;
+
+    &:hover {
+      background: $crm-blue-light;
+    }
+
+    > .add-icon {
+      background-image: none;
+      text-indent: 0;
+      margin-right: 3px;
+
+      &:after {
+        @include fa-icon($font-size-base, '\f067',$crm-white);
+        position: relative;
+        top: -1px;
+      }
+    }
+  }
+
+  table.vacancy-popup {
+    margin-bottom: 20px;
+  }
+
+  .view-hrvacancies {
+    .description {
+      margin-bottom: 20px;
+    }
+
+    td > a {
+      color: $brand-primary;
+    }
+  }
+
+  .CRM_HRRecruitment_Form_Search .crm-search-form-block {
+    margin-bottom: 20px;
+  }
+
+  #searchForm {
+    padding: 20px;
+
+    td {
+      padding: 0;
+    }
+
+    tr:nth-child(2) td {
+      padding: 20px 0;
+    }
+
+    #_qf_Search_refresh {
+      margin-bottom: 0 !important;
+    }
+
+    .listing-box {
+      border-color: $crm-copy;
+
+      > div {
+        background-color: transparent;
+
+        label {
+          color: $crm-copy;
+          font-weight: normal !important;
+          padding: 5px;
+        }
+      }
+    }
+
+    .font-size12pt {
+      font-size: $font-size-small;
+    }
+
+    .crm-form-text {
+      width: 312px;
+    }
+  }
+
+  &.page-civicrm-vacancy-dashboard .ui-dialog {
+    padding: 0;
+
+    &-content {
+      padding: 0;
+
+      .crm-block {
+        box-shadow: none;
+      }
+    }
+
+    table tr {
+      border-bottom: 0;
+
+      td {
+        padding-left: 0;
+        border-bottom: 0;
+      }
+    }
+
+    .advmultiselect tr:first-child td:nth-child(2) {
+      vertical-align: top;
+
+      input[type='button']:first-child {
+        margin-left: 0 !important;
+        margin-bottom: 15px !important;
+        width: 71px;
+        display: block;
+      }
+    }
+    
+    .ui-dialog-buttonpane {
+      margin: 0;
+    }
+  }
+
+  .hr-pipeline-tab {
+    .crm-accordion-header h2 {
+      color: $gray-darker !important;
+      display: inline-block;
+    }
+
+    .hr-vacancy {
+      width: 49%;
+      margin-bottom: 20px;
+      border: 1px solid $crm-background;
+      border-top-left-radius: 3px;
+      border-top-right-radius: 3px;
+      overflow: hidden;
+
+      > div {
+        padding: 0;
+        font-weight: normal;
+
+        > table {
+          width: 100%;
+          background-color: $crm-white;
+          border: 0;
+
+          tr {
+            &:nth-child(2) {
+              td {
+                line-height: 25px;
+              }
+            }
+
+            td {
+              color: $crm-dark-blue;
+              padding: 0 15px;
+
+              &:not(:first-child) {
+                padding: 10px 15px;
+              }
+
+              h3 {
+                background-color: $crm-copy;
+                padding: 8px 15px;
+
+                > a {
+                  color: $crm-white;
+
+                  .edit-icon {
```

Comment:
You have many nested levels. please try to reduce the hierarchy levels to 3/4 as a max